assets/scss/mixins/_typography.scss

```diff
@@ -0,0 +1,21 @@
+@mixin mobileToDesktopFontSize($fs: $fs-large, $fs-l: $fs-colossal) {
```

Comment:
I don't have any clue about `scss` but maybe is it worth to add some documentation for these methods. Just a oneliner telling what the mixing does is enough 👍